docs/reference/openapi.yaml

```diff
@@ -381,6 +395,18 @@ paths:
                 $ref: '#/components/schemas/EnvironmentResponse'
           description: Successful Response
       summary: Get Environment
+  /healthz:
```

Comment:
why `healthz` instead of `health` or `status` or similar?

Reply:
Sorry meant to say that it was convention but apparently I just sent a link without explanation.